cli/src/main.rs

```diff
@@ -192,10 +192,68 @@ fn run() -> Result<(), CliError> {
                                 .help("Path to private key file"),
                         )
                         .arg(
-                            Arg::with_name("path")
+                            Arg::with_name("node")
+                                .long("node")
                                 .takes_value(true)
                                 .required(true)
-                                .help("Path to a yaml file that defines the circuit proposal"),
+                                .multiple(true)
+                                .help(
```

Comment:
Please change this to long_help for better help formatting, same for all help that has multiple lines